### PR TITLE
Feature/inba 887 888 jf moment memcache

### PR DESCRIPTION
**Pull request by jmfadeley**
#### What does this PR do?
Updates moment.js to remove vulnerability, upgrades the Node version and removes memcache.

#### Related JIRA tickets:
INBA-887, INBA-888.

#### How should this be manually tested?
Bump your Node version up to or between 6.10.0 to 6.12.2. Run `yarn cache clean` and `yarn install`... just make sure that memcache is not in node_modules anymore. 

Run tests, regression test everything and be sure to do `node seed.js`. 

@ElijahGeorge please check that the kubernetes documentation is correct.

### CODE REVIEW

File: backend/test/util/shared-integration.js [hunk 1]
```diff
@@ -1,14 +1,12 @@
 'use strict';
 
-const sinon = require('sinon');
+// const sinon = require('sinon');
```
Comment (unprofessional) [inline backend/test/util/shared-integration.js]: Why comment instead of delete if we don't need it?

**Comment by Arghast** [comment]
@jmfadeley I followed the testing guidelines of this PR and blew away memcache from node_modules but am unable to start the Indaba backend due to the following memcache error.
image
Am I missing something?

**Comment by jmfadeley** [comment]
@Arghast I've been getting some strangeness where yarn is being lazy about actually blowing away the `node_modules`. Try `yarn install --force` to have it absolutely reinstall. Then make sure memcache isn't mentioned in the `yarn.lock` or in the `node_modules`.

**Comment by jmfadeley** [comment]
Several of the integration tests are broken. This was long on-going, even in develop. `user.integration.js` and `task.integration.js` work, the rest need a closer examination. I've gotten started on fixing them, but this calls for a new ticket, 889. Please rely on regression testing instead.

**Comment by Arghast** [comment]
@jmfadeley Alright, so when I attempt to run `yarn` `yarn install`, or `yarn install --force` I get this error
yarn install v1.7.0
image
And when I try running `node seed.js`, `yarn start` or `node app` I get this error
image
Am I missing something?

**Pull request merged**